.github/actions/bootstrap-poetry/action.yml

```diff
@@ -0,0 +1,34 @@
+name: Bootstrap Poetry
```

Comment:
Can we also use the cache action to cache `~/.cache/pypoetry/{artifacts,cache}` in this action till we can hopefully alter `setup-python` upstream?

Reply:
Actually scratch that we might not need it right now because the installs are really fast already with the default venv caching.

Reply:
Yes, but I want to keep that separate for now